kirstie/blog/blog-somanyreasons.md

```diff
@@ -0,0 +1,24 @@
+# So Many Reasons
```

Comment:
This looks like an interesting blog! I would love to help you work on this, or maybe you already published it?